design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+    shipping_discount_amount: Float @doc(description: "Shipping discount amount")
+    tax_amount: Float @doc(description: "Tax amount")
+    base_tax_amount: Float @doc(description: "base_tax_amount")
+    shipping_tax_amount: Float @doc(description: "shipping_tax_amount")
+    subtotal_incl_tax: Float @doc(description: "subtotal_incl_tax")
+    shipping_incl_tax: Float @doc(description: "shipping_incl_tax")
+    base_shipping_incl_tax: Float @doc(description: "base_shipping_incl_tax")
+    base_currency_code: String @doc(description: "base_currency_code")
+    quote_currency_code: String @doc(description: "quote_currency_code")
+    items_qty: Int @doc(description: "items_qty")
+    items: [ItemsOutput] @doc(description: "Items details")
+    total_segments: [TotalSegmentOutput] @doc(description: "Total segments")
+}
+
+type ItemsOutput {
+    item_id: ID @doc(description: "Item ID")
+    price: Float @doc(description: "Price")
+    base_price: Float @doc(description: "Base price")
+    qty: Int @doc(description: "Quantity")
+    row_total: Float @doc(description: "Row total")
+    base_row_total: Float @doc(description: "Base row total")
+    row_total_with_discount: Float @doc(description: "Row total with discount")
+    tax_amount: Float @doc(description: "Tax amount")
+    base_tax_amount: Float @doc(description: "Base tax amount")
+    tax_percent: Float @doc(description: "Tax percent")
+    discount_amount: Float @doc(description: "Discount amount")
+    base_discount_amount: Float @doc(description: "Base discount amount")
+    discount_percent: Float @doc(description: "Discount percent")
+    price_incl_tax: Float @doc(description: "Price incl. tax")
+    base_price_incl_tax: Float @doc(description: "Base price incl. tax")
+    row_total_incl_tax: Float @doc(description: "Row total incl. tax")
+    base_row_total_incl_tax: Float @doc(description: "Base row total incl. tax")
+    name: String @doc(description: "Name")
+}
+
+type TotalSegmentOutput {
+    code: String @doc(description: "Segment code")
+    title: String @doc(description: "Segment title")
+    value: Float @doc(description: "Segment value")
+}
+
+type TotalInfo {
+    totals: TotalsOutput! @doc(description: "List of order otals for negotiable quote")
+}
+
+input UpdatedPricesInput {
+    quote_ids: [String]! @doc(description: "IDs of the updated quotes")
+}
+
+input NegotiableQuoteRequestInput {
+    quote_id: String! @doc(description: "Quote hash")
+    quote_name: String @doc(description: "Quote name")
+    comment: String @doc(description: "Comment")
+    files: [FileInput] @doc(description: "Attached files")
+}
+
+type RequestQuoteOutput {
+    negotiable_quote:NegotiableQuote!
+}
+
+input FileInput @doc(description: "The list of file attachment codes") {
+    base64_encoded_data: String
+    type: String
+    name: String!
+    extension_attributes: [FileAttributeInput] @doc(description: "File extension attributes")
```

Comment:
What is the purpose of extension attributes here?

Reply:
removed the extension_attributes.